### PR TITLE
DO 217 Multiple UX Issue(15 May)

### PR DESCRIPTION
**Pull request by Palakagrawal28**
Update credentials button and form alignment.
Credential Popup.

### CODE REVIEW

File: src/components/routes/Member/AuthenticatedMember/UpdatePasswordRoute/_UpdatePassword.scss [hunk 1]
```diff
@@ -36,122 +106,106 @@
...
-        width: 50%;
-        margin: 0 auto;
-        border-radius: 30px;
-        margin-top: 50px;
-        background: #fff;
-        box-shadow: 0 0.46875rem 2.1875rem rgba(4, 9, 20, 0.03), 0 0.9375rem 1.40625rem rgba(4, 9, 20, 0.03), 0 0.25rem 0.53125rem rgba(4, 9, 20, 0.05), 0 0.125rem 0.1875rem rgba(4, 9, 20, 0.03);
-        @media(min-width: 1px) and (max-width: 1024px){
-            width: 80%;
-        }
+      //input {
```
Comment (amitjain24524) [inline src/components/routes/Member/AuthenticatedMember/UpdatePasswordRoute/_UpdatePassword.scss]: remove commented code

File: src/components/routes/Member/AuthenticatedMember/UpdatePasswordRoute/UpdatePassword.jsx [hunk 2]
```diff
@@ -54,7 +54,9 @@ class UpdatePassword extends React.Component {
   };
 
   renderButton = () => (
-    <Button className="login-button button__login" type="submit" value="Update" />
+    <div className={'update-credentials__button-container'}>
+      <Button className="update_button" type="submit" value="Update" />
```
Comment (amitjain24524) [inline src/components/routes/Member/AuthenticatedMember/UpdatePasswordRoute/UpdatePassword.jsx:58]: use button component

File: src/components/routes/Member/AuthenticatedMember/UpdatePasswordRoute/_UpdatePassword.scss [hunk 3]
```diff
@@ -1,32 +1,102 @@
...
+        background: #fff;
+        box-shadow: 0 0.46875rem 2.1875rem rgba(4, 9, 20, 0.03), 0 0.9375rem 1.40625rem rgba(4, 9, 20, 0.03), 0 0.25rem 0.53125rem rgba(4, 9, 20, 0.05), 0 0.125rem 0.1875rem rgba(4, 9, 20, 0.03);
       }
-      .form-error{
+      .form-error {
         top: 80px;
-        width: calc(50% - 30px);
+        width: auto;
         background: #c72727;
         height: 27px;
```
Comment (amitjain24524) [inline src/components/routes/Member/AuthenticatedMember/UpdatePasswordRoute/_UpdatePassword.scss]: why we are using fix height

File: src/components/routes/Member/AuthenticatedMember/UpdatePasswordRoute/_UpdatePassword.scss [hunk 4]
```diff
@@ -230,19 +255,22 @@
...
+  .updatePasswordMessage {
     padding: 30px;
     font-size: 24px;
     color: #000000;
     margin: 3px auto;
     text-align: center;
   }
-  button{
+
+  button {
```
Comment (amitjain24524) [inline src/components/routes/Member/AuthenticatedMember/UpdatePasswordRoute/_UpdatePassword.scss]: If you use button component no need to add CSS code for button.

File: src/components/routes/Member/AuthenticatedMember/UpdatePasswordRoute/UpdatePassword.jsx (hunk 2 above)
Comment (mohitsaladiya) [reply src/components/routes/Member/AuthenticatedMember/UpdatePasswordRoute/UpdatePassword.jsx:58]: It's already Button component.

File: src/components/routes/Member/AuthenticatedMember/UpdatePasswordRoute/UpdatePassword.jsx [hunk 5]
```diff
@@ -54,7 +54,9 @@ class UpdatePassword extends React.Component {
   };
 
   renderButton = () => (
-    <Button className="login-button button__login" type="submit" value="Update" />
+    <div className={'update-credentials__button-container'}>
```
Comment (mohitsaladiya) [inline src/components/routes/Member/AuthenticatedMember/UpdatePasswordRoute/UpdatePassword.jsx:57]: Why we are not using classNames as per the convention. It should be
```suggestion
    <div className={'update-password__button-container'}>
```

File: src/components/routes/Member/AuthenticatedMember/UpdatePasswordRoute/UpdatePassword.jsx (hunk 2 above)
Comment (mohitsaladiya) [inline src/components/routes/Member/AuthenticatedMember/UpdatePasswordRoute/UpdatePassword.jsx:58]:
```suggestion
      <Button className="update-password_button" type="submit" value="Update" />
```

File: src/components/routes/Member/AuthenticatedMember/UpdatePasswordRoute/UpdatePassword.jsx [hunk 6]
```diff
@@ -68,7 +70,7 @@ class UpdatePassword extends React.Component {
           isCloseButtonHidden
         >
           <div className={'updatePasswordMessage'}>{this.props.updatePasswordMessage}</div>
-          <Button value="Ok" onClick={this.redirectToLogout} />
+          <Button className={'updatePasswordMessage_button'} value="Ok" onClick={this.redirectToLogout} />
```
Comment (mohitsaladiya) [inline src/components/routes/Member/AuthenticatedMember/UpdatePasswordRoute/UpdatePassword.jsx:73]:
```suggestion
          <Button className={'update-password__message-button'} value="Ok" onClick={this.redirectToLogout} />
```

File: src/components/routes/Member/AuthenticatedMember/UpdatePasswordRoute/UpdatePassword.jsx [hunk 7]
```diff
@@ -82,37 +84,34 @@ class UpdatePassword extends React.Component {
...
-                  onSubmit={this.onSubmit}
-                  buttons={this.renderButton}
-                />
-              </div>
-              <div className={'BackToHomeButton'}>
-                <span>
-                  {this.renderBackToHomeButton()}
-                </span>
-              </div>
+      <div className={'update-credential-header'}>
```
Comment (mohitsaladiya) [inline src/components/routes/Member/AuthenticatedMember/UpdatePasswordRoute/UpdatePassword.jsx:87]:
```suggestion
      <div className={'update-password__header'}>
```

File: src/components/routes/Member/AuthenticatedMember/UpdatePasswordRoute/UpdatePassword.jsx [hunk 8]
```diff
@@ -82,37 +84,34 @@ class UpdatePassword extends React.Component {
...
+        <span className={'BackArrow'}>
+          {this.renderBackToHomeButton()}
+        </span>
+        <div className="update-password-container">
```
Comment (mohitsaladiya) [inline src/components/routes/Member/AuthenticatedMember/UpdatePasswordRoute/UpdatePassword.jsx:91]:
```suggestion
        <div className="update-password__container">
```

File: src/components/routes/Member/AuthenticatedMember/UpdatePasswordRoute/UpdatePassword.jsx [hunk 9]
```diff
@@ -82,37 +84,34 @@ class UpdatePassword extends React.Component {
...
+          <div className="update-password-container__form-container">
```
Comment (mohitsaladiya) [inline src/components/routes/Member/AuthenticatedMember/UpdatePasswordRoute/UpdatePassword.jsx:92]:
```suggestion
          <div className="update-password__form-container">
```

File: src/components/routes/Member/AuthenticatedMember/UpdatePasswordRoute/UpdatePassword.jsx [hunk 10]
```diff
@@ -82,37 +84,34 @@ class UpdatePassword extends React.Component {
...
+            <div className={'update-password-container__usernameForm'}>
```
Comment (mohitsaladiya) [inline src/components/routes/Member/AuthenticatedMember/UpdatePasswordRoute/UpdatePassword.jsx:93]:
```suggestion
            <div className={'update-password__usernameForm'}>
```

File: src/components/routes/Member/AuthenticatedMember/UpdatePasswordRoute/UpdatePassword.jsx [hunk 11]
```diff
@@ -82,37 +84,34 @@ class UpdatePassword extends React.Component {
...
+              {updatePasswordIsSuccess === false ?
+                <div className={'form-error'}><p>{updatePasswordMessage}</p></div> : null}
```
Comment (mohitsaladiya) [inline src/components/routes/Member/AuthenticatedMember/UpdatePasswordRoute/UpdatePassword.jsx:95]:
```suggestion
                <div className={'update-password__form-error'}><p>{updatePasswordMessage}</p></div> : null}
```

File: src/components/routes/Member/AuthenticatedMember/UpdatePasswordRoute/UpdatePassword.jsx [hunk 12]
```diff
@@ -82,37 +84,34 @@ class UpdatePassword extends React.Component {
...
+              <h3>{HEADER_TEXT.UPDATE_USERNAME}</h3>
+              <Form
+                formSchema={config.updateUsernameForm.formSchema}
+                formUISchema={config.updateUsernameForm.formUISchema}
+                onSubmit={this.onSubmit}
+                buttons={this.renderButton}
+              />
+            </div>
+            <div className={'update-password-container__passwordForm'}>
```
Comment (mohitsaladiya) [inline src/components/routes/Member/AuthenticatedMember/UpdatePasswordRoute/UpdatePassword.jsx:104]:
```suggestion
            <div className={'update-password__passwordForm'}>
```

File: src/components/routes/Member/AuthenticatedMember/UpdatePasswordRoute/_UpdatePassword.scss (hunk 1 above)
Comment (Palakagrawal28) [reply src/components/routes/Member/AuthenticatedMember/UpdatePasswordRoute/_UpdatePassword.scss]: Already Removed

File: src/components/routes/Member/AuthenticatedMember/UpdatePasswordRoute/UpdatePassword.jsx (hunk 2 above)
Comment (Palakagrawal28) [reply src/components/routes/Member/AuthenticatedMember/UpdatePasswordRoute/UpdatePassword.jsx:58]: Import button from typography elements

File: src/components/routes/Member/AuthenticatedMember/UpdatePasswordRoute/_UpdatePassword.scss (hunk 3 above)
Comment (Palakagrawal28) [reply src/components/routes/Member/AuthenticatedMember/UpdatePasswordRoute/_UpdatePassword.scss]: Already removed.

File: src/components/routes/Member/AuthenticatedMember/UpdatePasswordRoute/_UpdatePassword.scss (hunk 4 above)
Comment (Palakagrawal28) [reply src/components/routes/Member/AuthenticatedMember/UpdatePasswordRoute/_UpdatePassword.scss]: Already removed, when pushed.

File: src/components/routes/Member/AuthenticatedMember/UpdatePasswordRoute/UpdatePassword.jsx (hunk 5 above)
Comment (Palakagrawal28) [reply src/components/routes/Member/AuthenticatedMember/UpdatePasswordRoute/UpdatePassword.jsx:57]: DONE.

File: src/components/routes/Member/AuthenticatedMember/UpdatePasswordRoute/UpdatePassword.jsx (hunk 2 above)
Comment (Palakagrawal28) [reply src/components/routes/Member/AuthenticatedMember/UpdatePasswordRoute/UpdatePassword.jsx:58]: DONE.

File: src/components/routes/Member/AuthenticatedMember/UpdatePasswordRoute/UpdatePassword.jsx (hunk 6 above)
Comment (Palakagrawal28) [reply src/components/routes/Member/AuthenticatedMember/UpdatePasswordRoute/UpdatePassword.jsx:73]: DONE.

File: src/components/routes/Member/AuthenticatedMember/UpdatePasswordRoute/UpdatePassword.jsx (hunk 7 above)
Comment (Palakagrawal28) [reply src/components/routes/Member/AuthenticatedMember/UpdatePasswordRoute/UpdatePassword.jsx:87]: DONE.

File: src/components/routes/Member/AuthenticatedMember/UpdatePasswordRoute/UpdatePassword.jsx (hunk 8 above)
Comment (Palakagrawal28) [reply src/components/routes/Member/AuthenticatedMember/UpdatePasswordRoute/UpdatePassword.jsx:91]: DONE.

File: src/components/routes/Member/AuthenticatedMember/UpdatePasswordRoute/UpdatePassword.jsx (hunk 9 above)
Comment (Palakagrawal28) [reply src/components/routes/Member/AuthenticatedMember/UpdatePasswordRoute/UpdatePassword.jsx:92]: It is container inside another container ie. update-password__container is parent and inside that form-container that's why I've given it in same order.

update-password__container   //parent
update-password__container__form-container  //child

File: src/components/routes/Member/AuthenticatedMember/UpdatePasswordRoute/UpdatePassword.jsx (hunk 10 above)
Comment (Palakagrawal28) [reply src/components/routes/Member/AuthenticatedMember/UpdatePasswordRoute/UpdatePassword.jsx:93]: DONE.

File: src/components/routes/Member/AuthenticatedMember/UpdatePasswordRoute/UpdatePassword.jsx (hunk 12 above)
Comment (Palakagrawal28) [reply src/components/routes/Member/AuthenticatedMember/UpdatePasswordRoute/UpdatePassword.jsx:104]: DONE

File: src/components/routes/Member/AuthenticatedMember/UpdatePasswordRoute/UpdatePassword.jsx (hunk 11 above)
Comment (Palakagrawal28) [reply src/components/routes/Member/AuthenticatedMember/UpdatePasswordRoute/UpdatePassword.jsx:95]: DONE

File: src/components/routes/Member/AuthenticatedMember/UpdatePasswordRoute/UpdatePassword.jsx (hunk 9 above)
Comment (Palakagrawal28) [reply src/components/routes/Member/AuthenticatedMember/UpdatePasswordRoute/UpdatePassword.jsx:92]: Resolved as per your suggestion.

**Pull request merged**